README.md

```diff
@@ -139,14 +135,6 @@ The AWS IoT Device Defender library has no dependencies on additional libraries
 
 See memory requirements for the latest release [here](https://aws.github.io/aws-iot-device-sdk-embedded-C/202211.00/libraries/aws/device-defender-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#defender_memory_requirements).
 
-#### AWS IoT Over-the-air Update
```

Comment:
Lets keep this heading and talk about the removal since vX.X and recommend the transitioning to the modular OTA, please

Reply:
Done. Thanks.